srcpkgs/zramen/template

```diff
@@ -16,10 +16,6 @@ do_install() {
 	vbin zramen
 	vlicense UNLICENSE
 	vdoc README.md
-	vmkdir etc/sv
-	vcopy sv/zramen etc/sv
+	vsv sv/zramen
 	chmod 644 "${PKGDESTDIR}/etc/sv/zramen/conf"
```

Comment:
I believe this chmod can be removed as well.

Reply:
If upstream ships a tarball with conf with `755` perms, this should be done automatically by `vsv`.

Reply:
That's a good point, looks like `conf` has the right permissions here. I'm not sure that correcting permissions on `conf` is within the scope of `vsv` because a `conf` file is a common construct but subject to variations in individual services. The behavior of `run`, `finish` and `log/run` scripts are, however, directly specified in the `runit` docs. Either way, it is definitely outside the scope of this PR.